experimental/FTheoryTools/test/literature_models.jl

```diff
@@ -783,4 +783,44 @@ foah16_B3_weier = literature_model(arxiv_id = "1408.4808", equation = "3.203", t
   @test parent(explicit_model_sections(foah14_B3_weier)["s7"]) == cox_ring(base_space(foah14_B3_weier))
   @test parent(explicit_model_sections(foah15_B3_weier)["s7"]) == cox_ring(base_space(foah15_B3_weier))
   @test parent(explicit_model_sections(foah16_B3_weier)["s7"]) == cox_ring(base_space(foah16_B3_weier))
+  @test length(singular_loci(foah1_B3_weier)) == 1
+  @test length(singular_loci(foah2_B3_weier)) == 1
+  @test length(singular_loci(foah3_B3_weier)) == 1
+  @test length(singular_loci(foah4_B3_weier)) == 2
+  @test length(singular_loci(foah5_B3_weier)) == 1
+  @test length(singular_loci(foah6_B3_weier)) == 2
+  @test length(singular_loci(foah7_B3_weier)) == 1
+  @test length(singular_loci(foah8_B3_weier)) == 3
+  @test length(singular_loci(foah9_B3_weier)) == 2
+  @test length(singular_loci(foah10_B3_weier)) == 3
+  @test length(singular_loci(foah11_B3_weier)) == 3
+  @test length(singular_loci(foah12_B3_weier)) == 3
+  @test length(singular_loci(foah13_B3_weier)) == 4
+  @test length(singular_loci(foah14_B3_weier)) == 4
+  @test length(singular_loci(foah15_B3_weier)) == 5
+  @test length(singular_loci(foah16_B3_weier)) == 4
```

Comment:
For my curiosity - how long do you think those tests take now?

I believe we even raise a warning somewhere about `singular_loci` being potentially super slow. So I just want to gauge how much this slows down the OSCAR tests.

The tests are green - at least for now. That of course is a good indication.

Reply:
On my laptop, running all of the tests for FTheoryTools takes 6.5 minutes.

Reply:
That sounds good to me.